### PR TITLE
feat: improve slippage setting

### PR DESCRIPTION
**Pull request by TienPQ149**
Automatically adjust slippage setting to 0.1% when users are trying to swap stable token to stable token
Add another 0.05% slippage setting in our UI (currently we have 0.1%, 0.5%, 1%, Custom)
Always display Max Slippage in the Swap form - even when its 0.5% as default

### CODE REVIEW

File: src/pages/SwapV2/index.tsx [hunk 1]
```diff
@@ -665,6 +665,23 @@ export default function Swap({ history }: RouteComponentProps) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isExpertMode])
 
+  const [rawSlippage, setRawSlippage] = useUserSlippageTolerance()
+  const swapState = useSwapState()
+  const { INPUT, OUTPUT } = swapState
+
+  const isStableSwap =
```
Comment (hungdoansy) [inline src/pages/SwapV2/index.tsx:672]: I'd suggest this should be changed to `isStableCoinSwap`

**Review by hungdoansy — DISMISSED** [review]
LGTM

File: src/pages/SwapV2/index.tsx [hunk 2]
```diff
@@ -665,6 +665,23 @@ export default function Swap({ history }: RouteComponentProps) {
...
+    INPUT.currencyId &&
+    OUTPUT?.currencyId &&
+    chainId &&
+    STABLE_COINS_ADDRESS[chainId].includes(INPUT?.currencyId) &&
+    STABLE_COINS_ADDRESS[chainId].includes(OUTPUT?.currencyId)
+
+  useEffect(() => {
+    if (isStableSwap) {
+      setRawSlippage(10)
+    }
```
Comment (viet-nv) [inline src/pages/SwapV2/index.tsx:682]: wrong business logic here

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/pages/SwapV2/index.tsx [hunk 3]
```diff
@@ -665,6 +665,23 @@ export default function Swap({ history }: RouteComponentProps) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isExpertMode])
 
+  const [rawSlippage, setRawSlippage] = useUserSlippageTolerance()
+  const swapState = useSwapState()
+  const { INPUT, OUTPUT } = swapState
+
+  const isStableSwap =
+    INPUT.currencyId &&
```
Comment (nguyenhoaidanh) [inline src/pages/SwapV2/index.tsx:673]: INPUT?.currencyId

**Review by viet-nv — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**